### DISCUSSION TITLE
Casing for a MiniDexed module.

### DISCUSSION THREAD

**Discussion by Scorpiopraxis** [probonopd/MiniDexed#530] [Ideas]
Hi @probonopd,

I love your MiniDexed project. I'm from Dtronics in the Netherland. We made a programmer for the original Yamaha DX7 a few years back. It would be nice to have a Minidexed module next to the programmer.  So I have been drawing a metal casing for it. 
I'm goint to make it for a few friends that would love to have a Minidexed unit to play with. We made a extension unit with a display, outputs and controller.
I was thinking maybe we can make a DIY kit or a fully installed unit. For people that would like to use the MiniDexed unit.
If that's ok with you I have some questions , Can I use the project name "Minidexed" on the screenprint? Or something like "Inspired by github.com/probonopd"

Please let me know. We are doing this for fun so all the files Fusion360 / PCB is available to use.

Hope to get in touch with you...

![Screenshot 2023-08-10 212225](https://github.com/probonopd/MiniDexed/assets/141956839/23d72984-ed95-4eb8-98e3-6f02ccef9b23)
![Screenshot 2023-08-10 212208](https://github.com/probonopd/MiniDexed/assets/141956839/1dae8075-bd5d-478e-89c6-46edf30ca2c1)
![Screenshot 2023-08-10 212303](https://github.com/probonopd/MiniDexed/assets/141956839/79faa97a-5514-42b6-bc80-3e9a2ecc3c38)

**Comment by probonopd** [comment]
Hello @Scorpiopraxis, thanks for reaching out. I'm generally happy to support your project, please contact me at probono at puredarwin dot org. Thanks!
